### PR TITLE
[16.0][ADD] partner_export_wamas

### PR DESCRIPTION
**Pull request by QuocDuong1306**
Wait for `base_wamas_ubl`

### CODE REVIEW

File: partner_export_wamas/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        :return: partner data as WAMAS format
+        :rtype: a byte
+        """
+        self.ensure_one()
+        dict_partner = self.read()[0]
+        # If having a specific dict for the partner, we use it
+        if specific_dict and isinstance(specific_dict, dict):
+            dict_partner = specific_dict
+        base_wamas_ubl = self.env["base.wamas.ubl"]
+        return base_wamas_ubl.export_dict2wamas(dict_partner, telegram)
```
Comment (tuantrantg) [inline partner_export_wamas/models/res_partner.py]: - please replace `export_dict2wamas` by `dict2wamas`
- if there is no `telegram`, please raise warning missing telegram input

Comment (nilshamerlinck) [inline partner_export_wamas/models/res_partner.py]:
```suggestion
        return base_wamas_ubl.dict2wamas(dict_partner, telegram)
```

**Review by tuantrantg — CHANGES_REQUESTED** [review]

**Comment by QuocDuong1306** [comment]
Thank @nilshamerlinck @tuantrantg , I updated

File: partner_export_wamas/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    def _wamas_export(self, specific_dict=None, telegram=False):
+        """
+        Export the partner data as WAMAS format
+
+        :return: partner data as WAMAS format
+        :rtype: a byte
```
Comment (nilshamerlinck) [inline partner_export_wamas/models/res_partner.py]:
```suggestion
        :rtype: bytes
```

File: partner_export_wamas/tests/test_export_partner.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
+# Copyright 2023
```
Comment (nilshamerlinck) [inline partner_export_wamas/tests/test_export_partner.py]: see copyrights remark on channel

Comment (QuocDuong1306) [reply partner_export_wamas/tests/test_export_partner.py]: Thank @nilshamerlinck , I updated

**Pull request closed without merging**